### PR TITLE
feature/itemDetailPage

### PR DESCRIPTION
**Pull request by RomchikSt**
Trello: https://trello.com/c/PeLSAoql/14-item-detail-layout-refactoring

After:

https://github.com/unreal998/top-kolesa-fe/assets/140477189/b05d00c7-0f1b-4384-8e96-0d6641627263

### CODE REVIEW

File: src/modules/itemDetailsPage/components/Header.tsx [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+        fontFamily={FONTS.BOLD_TEXT_FAMILY}
+        fontWeight={600}
+        pb={2}
+        width={"fit-content"}
+      >
+        {selectedItemData?.brand} {selectedItemData?.name}{" "}
+        {selectedItemData?.width}/{selectedItemData?.height} R
+        {selectedItemData?.diametr}
+      </Typography>
+      <Box borderBottom={`1px dashed ${BASE_COLORS.DEFAULT_BLUE}`}></Box>
```
Comment (unreal998) [inline src/modules/itemDetailsPage/components/Header.tsx]: Replace to <Box … />

File: src/modules/itemDetailsPage/components/TireImg.tsx [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+import { Stack } from "@mui/material";
+
+import { selectSelectedItemData } from "../../shopPage/selectors";
+import { SHOP_ITEM_TIRES_IMG_PREFIX } from "../../../constants";
+
+export default function TireImg() {
+  const selectedItemData = useSelector(selectSelectedItemData());
+
+  return (
+    <Stack
```
Comment (unreal998) [inline src/modules/itemDetailsPage/components/TireImg.tsx]: Move to the separate styled component

File: src/modules/itemDetailsPage/types.tsx [hunk 3]
```diff
@@ -1,71 +1,71 @@
...
+  smsText: string;
+  smsSendAt: string;
+};
 
 export const PRODUCT_TYPE = {
-    TIRE: 'TIRE',
-    DISK: 'DISK'
-} 
\ No newline at end of file
+  TIRE: "TIRE",
```
Comment (unreal998) [inline src/modules/itemDetailsPage/types.tsx]: use a single quotes style

File: src/modules/shopPage/reducer.tsx [hunk 4]
```diff
@@ -27,6 +27,7 @@ export type ShopItemAPI = {
   brand_id: number;
   size_id: number;
   supplier_id: number;
+  param: string;
```
Comment (unreal998) [inline src/modules/shopPage/reducer.tsx]: What that “param” for ?

**Review by unreal998 — APPROVED** [review]
See my comments

File: src/modules/shopPage/reducer.tsx (hunk 4 above)
Comment (RomchikSt) [reply src/modules/shopPage/reducer.tsx]: param - its data where we can see tires have or haven't studded and I need add it if I want see use this data
I use it in Characteristics component

File: src/modules/itemDetailsPage/components/TireImg.tsx (hunk 2 above)
Comment (RomchikSt) [reply src/modules/itemDetailsPage/components/TireImg.tsx]: done

File: src/modules/itemDetailsPage/components/Header.tsx (hunk 1 above)
Comment (RomchikSt) [reply src/modules/itemDetailsPage/components/Header.tsx]: done

File: src/modules/itemDetailsPage/types.tsx (hunk 3 above)
Comment (RomchikSt) [reply src/modules/itemDetailsPage/types.tsx]: done

File: src/modules/shopPage/reducer.tsx (hunk 4 above)
Comment (RomchikSt) [reply src/modules/shopPage/reducer.tsx]: added note

**Pull request merged**